### ISSUE TITLE
Android Oreo stable build, not working

### ISSUE THREAD

**Issue by samshiver** [Deletescape-Media/Lawnchair#568]
I've just installed the stable build of Android 8.0.0 on my Nexus 5X and Lawnchair doesn't even start after a fresh install. I set it to default launcher and it just shows an empty homescreen and I can't do anything but open the phone settings through the status bar and change the default launcher.
I'd also want to add that it used to work after the android upgrade, maybe because it was already running on Nougat (although it was very laggy, way more behind than the smooth experience on android 7.1.2) but, as I said before, after a hard reset of the phone and a fresh install of Lawnchair, the launcher doesn't event start. It needs to be fixed for Android O

**Comment by KL3N** [comment]
I have same problem. Thought it was something to do with the rooted tweeks I'd made, but after full unroot and factory reset I still have issue

**Comment by AxiK101** [comment]
Probably not compatible yet

_Sent from my Sony F3211 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by KL3N** [comment]
I've been using lawnchair on all the beta versions of Android o, not one issue until the stable version

**Comment by jjanku** [comment]
I can confirm this issue too using my Nexus 5X with stable Android Oreo.

Fresh install of the app and granting all permissions (storage, phone, location) in settings before starting the launcher fixed it for me, give it a try.

Has anyone tried building the alpha version to see whether the bug has been resolved since the last stable release?

**Comment by edwaine** [comment]
Same happened to me on the Nexus 5X but not on my Pixel XL (I wanna use the Round Icons on all apps).

**Comment by nyancrimew** [comment]
@jjanku you can download an alpha version from https://lawnchairmirror.duckdns.org to try this out yourself

**Comment by edwaine** [comment]
@deletescape Lawnchair-1.1.0.1186.apk didn't work on my Nexus 5X. Now I'm trying the lawnchair-latest.apk->> This one worked.

**Comment by nyancrimew** [comment]
So this is basically fixed already, but not in our currently stable version.

**Comment by jjanku** [comment]
@deletescape the lawnchair-latest.apk kinda works, I don't think the issue is resolved though.
After reinstalling it a couple of times, sometimes it loads right away, sometimes not.

Part of logcat (probably useless):
31662-31662/? E/OverlaySController: Calling with an invalid UID, the interface will not work
31662-31662/? E/OverlaySController: Invalid version
2174-2174/ch.deletescape.lawnchair I/Process: Sending signal. PID: 2174 SIG: 9
3539-31655/? I/ActivityManager: Process ch.deletescape.lawnchair (pid 2174) has died: fore TOP 
3539-31655/? I/ActivityManager: Start proc 2198:ch.deletescape.lawnchair/u0a177 for activity ch.deletescape.lawnchair/.Launcher

@edwaine when you reinstall it, does it always work for you?

**Comment by edwaine** [comment]
@jjanku I did reinstall it so yes, I think that method works for me.

**Comment by ArmanKir-yan25** [comment]
Stupids clear all data and force stop

**Comment by GroovinChip** [comment]
@edwaine Sorry about the thumbs up and down on your comment - accident on FastHub!

_Sent from my Google Pixel XL using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by KL3N** [comment]
I've got v1.0.1063 to work, but it no longer has adaptive icon options in settings. Is this issue same with others? Have you found a solution, or should I open it as a new issue?
